src/main/java/duke/command/HelpCommand.java

```diff
@@ -1,43 +1,43 @@
-package duke.command;
-
-import duke.core.DukeException;
-import duke.patient.PatientManager;
-import duke.storage.PatientStorage;
-import duke.storage.PatientTaskStorage;
-import duke.storage.TaskStorage;
-import duke.relation.PatientTaskList;
-import duke.core.Ui;
-import duke.task.TaskManager;
-
-public class HelpCommand extends Command {
-
-    /**
-     * .
-     *
-     * @param patientTask        .
-     * @param tasks              .
-     * @param patientList        .
-     * @param ui                 .
-     * @param patientTaskStorage .
-     * @param taskStorage        .
-     * @param patientStorage     .
-     * @throws DukeException .
-     */
-    @Override
-    public void execute(PatientTaskList patientTask, TaskManager tasks, PatientManager patientList,
-                        Ui ui, PatientTaskStorage patientTaskStorage, TaskStorage taskStorage,
-                        PatientStorage patientStorage) throws DukeException {
-        //ui.showHelpCommand();
-
-    }
-
-    /**
-     * .
-     *
-     * @return .
-     */
-    @Override
-    public boolean isExit() {
-        return false;
-    }
-}
+//package duke.command;
+//
+//import duke.core.DukeException;
+//import duke.patient.PatientManager;
+//import duke.storage.PatientStorage;
+//import duke.storage.PatientTaskStorage;
+//import duke.storage.TaskStorage;
+//import duke.relation.PatientTaskList;
+//import duke.core.Ui;
+//import duke.task.TaskManager;
+//
+//public class HelpCommand extends Command {
+//
+//    /**
+//     * .
+//     *
+//     * @param patientTask        .
+//     * @param tasks              .
+//     * @param patientList        .
+//     * @param ui                 .
+//     * @param patientTaskStorage .
+//     * @param taskStorage        .
+//     * @param patientStorage     .
+//     * @throws DukeException .
+//     */
+//    @Override
+//    public void execute(PatientTaskList patientTask, TaskManager tasks, PatientManager patientList,
+//                        Ui ui, PatientTaskStorage patientTaskStorage, TaskStorage taskStorage,
+//                        PatientStorage patientStorage) throws DukeException {
+//        //ui.showHelpCommand();
+//
+//    }
+//
+//    /**
+//     * .
+//     *
+//     * @return .
+//     */
+//    @Override
+//    public boolean isExit() {
+//        return false;
+//    }
+//}
```

Comment:
This is an non-related changes belong to other feature. It shouldnt be modified in this PR for review readability

Reply:
Since the help command is not in use , so i commented them out first

Reply:
It does not make sense as it is not belong to this PR. 
This PR should only involve file changes that belong to the FreqCounter.
A PR should only serve one purpose.